snap/info.go

```diff
@@ -997,3 +997,28 @@ func DropNick(nick string) string {
 	}
 	return nick
 }
+
+// StoreName splits the instance name and returns the store name of the snap.
+func StoreName(name string) string {
+	return strings.SplitN(name, "_", 2)[0]
+}
+
+// SplitInstanceName splits the instance name and returns the store name and the
+// instance key.
+func SplitInstanceName(name string) (store string, instanceKey string) {
+	split := strings.SplitN(name, "_", 2)
+	store = split[0]
+	if len(split) > 1 {
+		instanceKey = split[1]
+	}
+	return store, instanceKey
+}
+
+// InstanceName takes the store name and the instance key and returns an instance
+// name of the snap.
+func InstanceName(storeName string, instanceKey string) string {
```

Comment:
Nice :)

Reply:
could also make it `(storeName, instanceKey string)` like above :-)